src/components/GRTFeedbackView/index.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { DateTime } from 'luxon';
+// eslint-disable-next-line camelcase
+import { GetGRTFeedback_grtFeedbacks } from 'queries/types/GetGRTFeedback';
+
+import HelpText from 'components/shared/HelpText';
+
+type GRTFeedbackViewProps = {
+  // eslint-disable-next-line camelcase
+  grtFeedbacks: GetGRTFeedback_grtFeedbacks[];
+};
+
+const GRTFeedbackView = ({ grtFeedbacks }: GRTFeedbackViewProps) => {
+  return (
+    <>
+      <h2 className="margin-bottom-3 margin-top-0">Recommendations</h2>
+      {grtFeedbacks.some(grtFeedback => grtFeedback.feedbackType === 'GRB') && (
+        <div>
+          <h3 className="margin-bottom-1">GRT recommendations to the GRB</h3>
+          <HelpText className="margin-bottom-2">
+            These are the Governance Review Team recommendations for the
+            Governance Review Board.
+          </HelpText>
+          {grtFeedbacks
+            .filter(grtFeedback => grtFeedback.feedbackType === 'GRB')
+            .sort(
+              (a, b) =>
+                DateTime.fromISO(a.createdAt).toMillis() -
+                DateTime.fromISO(b.createdAt).toMillis()
+            )
+            .map(grtFeedback => {
+              return (
+                <div>
+                  <div className="text-bold">
+                    {DateTime.fromISO(grtFeedback.createdAt).toLocaleString(
+                      DateTime.DATE_MED
+                    )}
+                  </div>
+                  <p className="margin-bottom-3 margin-top-1 line-height-body-3">
+                    {grtFeedback.feedback}
+                  </p>
+                </div>
+              );
+            })}
+        </div>
+      )}
+      {grtFeedbacks.some(
```

Comment:
I think there's some clean up we can do here to avoid some of these loops. You can store the filtered `grtFeedbacks` that match the correct `feedbackType` in a variable. Then you can use that variable to decide whether there's any feedback and use iterate over that down below.